### PR TITLE
Create migration_task template

### PR DESCRIPTION
**Pull request by taquitos**
For use for the Swift migration related milestones and project board. It will make creating and linking stories easier 🚀

### CODE REVIEW

**Comment by taquitos** [comment]
Fixes https://github.com/RevenueCat/purchases-ios/issues/535

File: .github/ISSUE_TEMPLATE/migration_task [hunk 1]
```diff
@@ -0,0 +1 @@
+Please add screenshots, links to code, any relevant information, and then add any relevant tag, e.g.:  "Test backfill" for when more tests are needed, or "you can do this" for things that don't need any extra context and are good issues for new folks to help out with.
```
Comment (aboedo) [inline .github/ISSUE_TEMPLATE/migration_task:1]: should this file's extension be `.md`?
Also, perhaps line breaks would make it easier to read in editors that don't break lines automatically?

Comment (taquitos) [reply .github/ISSUE_TEMPLATE/migration_task:1]: YESSSSSSSS

Comment (taquitos) [reply .github/ISSUE_TEMPLATE/migration_task]: ✅

**Review by aboedo — APPROVED** [review]
🎉🎉🎉

**Pull request merged**